src/components/ResultsView.tsx

```diff
@@ -662,6 +691,8 @@ const ResultsView: ResultsViewComponent = ({
                 pageSizeTimeoutRef={pageSizeTimeoutRef}
                 onRefresh={onRefresh}
                 allResultsLength={allResults.length}
+                minimal={layout.mode === "minimal"}
+                showInterface={showInterface}
```

Comment:
This makes me think that this could be layout settings for `table` instead of a separate layout

Reply:
While I'm hesitant to add another menu item for a single layout and a single setting, I agree with you, as it will allow for further settings per layout in the future 👍

Reply:
The concept of layout settings I'm starting in the kanban PR #71, taking inspiration for how Notion does layout settings which works rather well